WordPress/src/main/java/org/wordpress/android/ui/media/MediaBrowserActivity.java

```diff
@@ -516,6 +516,8 @@ public boolean onMenuItemActionCollapse(MenuItem item) {
 
         enableTabs(true);
 
+        reloadMediaGrid();
```

Comment:
Since we'll handle the search behavior in https://github.com/wordpress-mobile/WordPress-Android/issues/7398, we should probably remove this addition to keep the changes in their respective pull request.

Reply:
Done!